src/main/java/com/yahoo/sketches/theta/ConcurrentUpdateSketch.java

```diff
@@ -0,0 +1,83 @@
+package com.yahoo.sketches.theta;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author eshcar
+ */
+public class ConcurrentUpdateSketch extends UpdateSketchComposition {
+
+  private static int PARALLELISM_LEVEL = 3;
```

Comment:
I would be nice if you explained in a code comment what PARALLELISM_LEVEL = 3 means and why 3 was chosen.

Reply:
I agree.
Basically this is the number of threads allocated to run propagation from local sketches to shared sketches in the background.
3 is not carved in stone, maybe it should be configurable, but we should be able to set a default value based on benchmarks.

Reply:
Yes, configurable and a default.  It is hard for us to predict the environment where this will be used.